mlflow/shap.py

```diff
@@ -32,3 +39,70 @@ def _log_matplotlib_figure(fig, out_file, artifact_path=None):
     """
     with _log_artifact_contextmanager(out_file, artifact_path) as tmp_path:
         fig.savefig(tmp_path)
+
+
+@experimental
+def log_explanation(predict_function, features, artifact_path=None):
+    """
+    Log the following items generated by SHAP.
+
+        - base values
+        - shap values
+        - summary bar plot
```

Comment:
```suggestion
        - summary bar plot (shows the average impact of each feature on model output)
```

Could we add brief descriptions like this for each of the logged artifacts to help the users understand them?